specs/toolsets/cmake/setup-toolset.sh

```diff
@@ -4,17 +4,19 @@
 export HAM_TOOLSET=CMAKE
 export HAM_TOOLSET_NAME=cmake
 export HAM_TOOLSET_DIR="${HAM_HOME}/toolsets/cmake"
+export HAM_CMAKE_HOME="${HAM_HOME}/toolsets/cmake"
+export PATH="${HAM_TOOLSET_DIR}:${PATH}"
 
 # path setup
 case $HAM_OS in
     NT*)
-        toolset_check_and_dl_ver cmake nt-x86 v1 || return 1
         export CMAKE_DIR="${HAM_TOOLSET_DIR}/nt-x86/"
         export PATH=${CMAKE_DIR}/bin:${PATH}
         if [ ! -e "$CMAKE_DIR/bin/cmake.exe" ]; then
             echo "E/cmake.exe doesn't exist in the toolset"
             return 1
         fi
+        . ham-toolset-import.sh msvc_15_x64
```

Comment:
Mmm? Is clang in that toolset? Also now msvc_15_x64 is deprecated, should be msvc_19_x64.